spec/controllers/graphql/queries/norm_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Graphql
+  RSpec.describe GraphqlController, type: :controller do
+    subject(:query_freaks) { post :execute, params: params, as: :json }
+
+    let(:params) do
+      {
+        query: File.read('spec/fixtures/requests/queries/norms.graphql')
+      }
+    end
+
+    it { is_expected.to match_response_for(queries: :norms, sample: :default) }
```

Comment:
How are the norms populated for the test ?

Reply:
i need to put query instead of queries